### PR TITLE
Created Locale for en-GH

### PR DESCRIPTION
**Pull request by wocodes**
Created test case for en-GH

Updated README file

### CODE REVIEW

**Comment by mastermunj** [comment]
@wocodes Thank you for the PR.

Please help resolve the issues with tests.

**Comment by wocodes** [comment]
@mastermunj This has been updated

File: package-lock.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "to-words",
   "version": "2.4.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (mastermunj) [inline package-lock.json:4]: @wocodes We have moved on to lockfile version 2. You'd need to either remove package-lock.json from commits or upgrade your npm to ensure correct changes come in.

**Review by mastermunj — CHANGES_REQUESTED** [review]

**Comment by wocodes** [comment]
@mastermunj Thanks for the heads up. I've updated this PR. Kindly review

**Comment by mastermunj** [comment]
Thank you!

**Pull request merged**